commercial/app/model/commercial/books/BookFinder.scala

```diff
@@ -72,6 +72,9 @@ object MagentoService extends ExecutionContexts with Logging {
     )
   }
 
+  private implicit val singleThreadExecutionContext: ExecutionContext =
+    Akka.system.dispatchers.lookup("play.akka.actor.single-thread")
```

Comment:
It's being used for a very specific purpose at the moment and only on Commercial servers so probably sensible to rename.